web/packages/plugins/default-pdf/src/PDFViewer.tsx

```diff
@@ -31,6 +36,7 @@ export const ViewerPDFPlugin = (props: any) => {
   const [blobUrl, setBlobUrl] = useState('')
   const [loading, setLoading] = useState(true)
   const [error, setError] = useState<string | null>(null)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore